### PR TITLE
fixed issue with commit #8 - adding Id and Class to html tags

### PR DESCRIPTION
**Pull request by WaydeTheKiwi**
Added class ids to all input options and divs etc for custom CSS
If Null, sets to default
formID
formClass
footerClass
headerClass
submitClass
inputClass
divGroupClass
divGroupTitleClass
divInputGroupClass
divSelectTitle
divformSelect

### CODE REVIEW

**Comment by ItsAegeus** [comment]
Can confirm it fixed the problems.

**Comment by WaydeTheKiwi** [comment]
It did for me, go test it

File: html/script.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 let formInputs = {};
+const formID = "";
```
Comment (IdrisDose) [inline html/script.js:2]: Should be changed to `let formID = "";`

File: html/script.js [hunk 2]
```diff
@@ -9,13 +10,13 @@ const OpenMenu = (data) => {
     $(`.main-wrapper`).fadeIn(0);
 	
 	const formID = data.formID != null ? data.formID : "qb-input-form";
```
Comment (IdrisDose) [inline html/script.js:12]: remove `const` keyword and change to `formID = data.formID != null ? data.formID : "qb-input-form";`

File: html/script.js [hunk 3]
```diff
@@ -56,15 +57,16 @@ const OpenMenu = (data) => {
 
     $(".main-wrapper").html(form.join(" "));
 
-    $("#qb-input-form").on("change", function (event) {
+    $(formID).on("change", function (event) {
```
Comment (IdrisDose) [inline html/script.js:60]: Will create issues;

try changing to $(\`#${formID}\`).on("change"

File: html/script.js [hunk 4]
```diff
@@ -56,15 +57,16 @@ const OpenMenu = (data) => {
...
         formInputs[$(event.target).attr("name")] = $(event.target).val();
     });
 
-    $("#qb-input-form").on("submit", async function (event) {
+    $("#" + formID).on("submit", async function (event) {
```
Comment (IdrisDose) [inline html/script.js:64]: please try and stick to string literal rather than string concats:
$(\`#${formID}\`)

File: html/script.js [hunk 5]
```diff
@@ -56,15 +57,16 @@ const OpenMenu = (data) => {
...
+		
         if (event != null) {
             event.preventDefault();
         }
-        let formData = $("#qb-input-form").serializeArray();
+        let formData = $("#" + formID).serializeArray();
```
Comment (IdrisDose) [inline html/script.js:69]: please try and stick to string literal rather than string concats:
$(\`#${formID}\`)

File: html/script.js [hunk 6]
```diff
@@ -170,7 +172,7 @@ const renderSelectInput = (item) => {
 
 const CloseMenu = () => {
     $(`.main-wrapper`).fadeOut(0);
-    $("#qb-input-form").remove();
+    $(formID).remove();
```
Comment (IdrisDose) [inline html/script.js:175]: Will also not run since wrong target.
$(\`#${formID}\`)

**Review by IdrisDose — CHANGES_REQUESTED** [review]
I've written comments on the code where there will be problems.

**Pull request closed without merging**